src/microbe_stage/Microbe.cs

```diff
@@ -2108,6 +2110,137 @@ private Transform GetNewPhysicsRotation(Transform transform)
+    private void ApplyPhysicsRotation(PhysicsDirectBodyState physicsState)
+    {
+        var transform = physicsState.Transform;
+        var target = transform.LookingAt(LookAtPoint, new Vector3(0, 1, 0));
+        if (transform == target)
+            return;
+
+        // Strength can later be affected by other values
+        // More strength is required to rotate larger microbes
+        var strength = Constants.CELL_BASE_THRUST * (1.0f + HexCount * 0.8f);
+
+        // Calculate the distance to reach the target rotation in degrees
+        var distance = target.basis.GetEuler().y - transform.basis.GetEuler().y;
+        distance = (float)(distance * (180.0f / Math.PI));
+        if (distance > 180.0f)
+        {
+            distance = -360.0f + distance;
+        }
+        else if (distance < -180.0f)
+        {
+            distance = 360.0f + distance;
+        }
+
+        var force = 0.0f;
+
+        // Angular velocity of the microbe
+        var velocity = physicsState.AngularVelocity.y;
+
+        // Maximum desired angular velocity
+        var maxVelocity = 5.0f;
+
+        // Brake if velocity exceeds maxVelocity
+        if (velocity < -maxVelocity)
+        {
+            force = 1.0f;
+        }
+        else if (velocity > maxVelocity)
+        {
+            force = -1.0f;
+        }
+        else
+        {
+            // Velocity in a range from -90(-maxVelocity) to 90(maxVelocity)
+            var velocityTo90 = 0.0f;
+            if (velocity != 0.0f)
+                velocityTo90 = velocity / maxVelocity * 90.0f;
+
+            // What to do if velocity is nearly zero
+            if (velocityTo90 > -0.1f && velocityTo90 < 0.1f)
+            {
+                // Rotate the microbe if it is not near the target rotation
+                if (distance > 2.0f)
+                {
+                    force = 1.0f;
+                }
+                else if (distance < -2.0f)
+                {
+                    force = -1.0f;
+                }
+            }
+
+            // Accelerate to maxVelocity if distance > 90
+            // Use only 95% of maxVelocity to not exceed it
+            else if (distance >= 90 && velocity < maxVelocity * 0.95f)
+            {
+                force = 1.0f;
+            }
+            else if (distance <= -90 && velocity > -maxVelocity * 0.95f)
+            {
+                force = -1.0f;
+            }
+
+            // If microbe is rotating left
+            else if (velocityTo90 > 0.0f)
+            {
+                // Brake if rotating to fast
+                if (distance > 0.0f && distance < 90.0f && velocityTo90 / distance > 0.7f)
+                {
+                    force = -(velocityTo90 / distance) * 1.5f;
+                }
+
+                // Accelerate if rotating to slow
+                else if (distance > 1.5f && distance < 90.0f && velocityTo90 / distance < 0.7f)
+                {
+                    force = distance / velocityTo90;
```

Comment:
I'm sorry, but I am very disturbed by the fact that a force is a speed over a distance, or a distance over a speed here. Perhaps in the first case 1.5f actually has units, but then it should appear somehow (e.g. by the constant's name, as it should also be defined as one).